dataframe-csv/src/main/kotlin/org/jetbrains/kotlinx/dataframe/documentation/DelimParams.kt

```diff
@@ -0,0 +1,245 @@
+     *
+     *   Useful for files with metadata, or comments at the beginning, or to give a custom \[header\].
+     */
+    const val SKIP_LINES: Long = 0L
+
+    /**
+     * @param readLines The maximum number of lines to read from the data. Default: `null`.
+     *
+     *   If `null`, all lines will be read.
+     */
+    val READ_LINES: Long? = null
+
+    /**
+     * @param parserOptions Optional [parsing options][ParserOptions] for columns initially read as [String].
+     *   Default, [DEFAULT_PARSER_OPTIONS]:
+     *
+     *   [ParserOptions][ParserOptions]`(`
+     *
+     *   {@include [Indent]}[nullStrings][ParserOptions.nullStrings]`  =  `{@include [DefaultNullStringsContentLink]}`,`
+     *
+     *   {@include [Indent]}[useFastDoubleParser][ParserOptions.useFastDoubleParser]` = true,`
+     *
+     *   `)`
+     *
+     *   Can configure locale, date format, double parsing, skipping types, etc.
+     *
+     *   **NOTE:** Make sure to use [DEFAULT_PARSER_OPTIONS][DEFAULT_PARSER_OPTIONS]`.copy()` to override the desired options.
+     */
+    val PARSER_OPTIONS: ParserOptions = DEFAULT_PARSER_OPTIONS
+
+    /**
+     * @param ignoreEmptyLines Whether to skip intermediate empty lines. Default: `false`.
+     *
+     *   If `false`, empty lines will be interpreted as having _empty_ values if \[allowMissingColumns\].
+     */
+    const val IGNORE_EMPTY_LINES: Boolean = false
+
+    /**
+     * @param allowMissingColumns Wether to allow rows with fewer columns than the header. Default: `true`.
+     *
+     *   If `true`, rows that are too short will be interpreted as _empty_ values.
+     */
+    const val ALLOW_MISSING_COLUMNS: Boolean = true
+
+    /**
+     * @param ignoreExcessColumns Whether to ignore rows with more columns than the header. Default: `true`.
+     *
+     *   If `true`, rows that are too long will have those columns dropped.
+     */
+    const val IGNORE_EXCESS_COLUMNS: Boolean = true
+
+    /**
+     * @param quote The quote character. Default: `"`.
+     *
+     *   Used when field- or line delimiters should be interpreted as literal text.
+     *
+     *   For example: `123,"hello, there",456,` would correspond to: `123`; `hello, there`; `456`.
+     */
+    const val QUOTE: Char = '"'
+
+    /**
+     * @param ignoreSurroundingSpaces Whether to ignore leading and trailing blanks around non-quoted fields.
+     *   Default: `true`.
+     */
+    const val IGNORE_SURROUNDING_SPACES: Boolean = true
+
+    /**
+     * @param trimInsideQuoted Whether to ignore leading and trailing blanks inside quoted fields.
+     *   Default: `false`.
+     */
+    const val TRIM_INSIDE_QUOTED: Boolean = false
+
+    /**
+     * @param parseParallel Whether to parse the data in parallel. Default: `true`.
+     *
+     *   If `true`, the data will be parsed in parallel.
+     *   This is usually faster, but can be turned off for debugging.
+     */
+    const val PARSE_PARALLEL: Boolean = true
```

Comment:
i feel it's a big ambiguous. So it's not about deephaven parser, rather it's about DataColumn.parse?

Reply:
no it's deephaven, our parser is not parallel atm. I'll specify.